src/common/env.ts

```diff
@@ -7,3 +7,6 @@ export const isWindowsOS = (process.platform === 'win32');
 export const isLinux = (process.platform === 'linux');
 
 export const isNodeEnv = !!process.env.NODE_ENV;
+
+export const isDemoEnv = process.env.ELECTRON_DEMO ?
```

Comment:
I think it is better to go with `isDevEnv` instead of creating a new one specific to this use case

Reply:
But we don't want it to be done when we do "npm run dev", right? Then we want the standard behaviour for origin. Only if we do "npm run demo mac" do we want the '*' origin, yes?